testing/testdata/testmodels2.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto2";
```

Comment:
Let's move all of these to `internal/testdata`, so the resulting Go code is not part of the exported API of this repo.

Reply:
This is confusing that you have two test files with "2" and "3" suffix. What happened to number one? They should both be combined into `testdata.proto`. OR you could just use well-known types (that are part of the protobuf runtime library) and not even bother with custom messages here.